PerformanceTests/DecoderTest/Configurations/Base.xcconfig

```diff
@@ -120,6 +120,7 @@ DEAD_CODE_STRIPPING_normal = YES;
 DEAD_CODE_STRIPPING = $(DEAD_CODE_STRIPPING_$(CURRENT_VARIANT));
 
 SDKROOT = macosx.internal;
+ADDITIONAL_SDKS = $(WK_ADDITIONAL_SDKS);
```

Comment:
That is the reason to indirectly define `ADDITIONAL_SDKS` via `WK_ADDITIONAL_SDKS`? Can we define `ADDITIONAL_SDKS` in `SDKAdditions.xcconfig`?

Also, while this avoids the need to inherit from Base to DebugRelease that was discussed earlier, are there any other sources to inherit from, like command line perhaps?